app/models/manageiq/providers/vmware/infra_manager/vm.rb

```diff
@@ -9,6 +9,8 @@ class ManageIQ::Providers::Vmware::InfraManager::Vm < ManageIQ::Providers::Infra
     unsupported_reason_add(:clone, _('Clone operation is not supported')) if blank? || orphaned? || archived?
   end
 
+  supports :reconfigure_disks
```

Comment:
there should be a `validate_reconfigure_disks` somewhere in the code if this is not a new feature